repl/appengine/web/src/app/reference_panel/reference-panel-component.html

```diff
@@ -16,8 +16,46 @@
 
 <h1>References</h1>
 
-<p>See <a href="https://github.com/google/cel-spec">github.com/google/cel-spec</a> for CEL language
-    overview. </p>
-<p>See
-    <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
-    for REPL syntax guide.</p>
\ No newline at end of file
+<mat-accordion>
+    <mat-expansion-panel>
+        <mat-expansion-panel-header>
+        <mat-panel-title>
+            CEL Specification
+        </mat-panel-title>
+        </mat-expansion-panel-header>
+        <p>See <a href="https://github.com/google/cel-spec">github.com/google/cel-spec</a> for CEL language
+            overview.</p>
+    </mat-expansion-panel>
+    <mat-expansion-panel>
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+            REPL Syntax
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <p>See <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
+            for REPL syntax guide.</p>
+    </mat-expansion-panel>
+    <mat-expansion-panel>
```

Comment:
Consider expanding the panels by default
```suggestion
    <mat-expansion-panel [expanded]="true">
```